### DISCUSSION TITLE
Anyone using espanso with wezterm?

### DISCUSSION THREAD

**Discussion by AndydeCleyre** [espanso/espanso#1329] [General]
Hello! Is anyone else using espanso with wezterm? 

I don't know if a certain configuration can avoid this, or if it's only happening to me, but here on Gnome+pop!_os+wezterm, the replacements don't successfully *shift* keys; that is, what should be underscores come out as dashes, ampersands come out as sevens, at symbols come out as twos, colons as semicolons, etc.

I'm curious if anyone has any corroborating or contradictory experience, before I bother opening an issue. I also don't plan to stay on gnome much longer so if it's particular to that I may not pursue the problem further.

**Comment by dodalovicgran** [comment]
for me espanso doesn't play well with the wezterm

**Reply by AndydeCleyre** [reply]
Thanks! Does the problem take the same form, acting as if the shift key is never pressed? And are you also using Linux, or another platform?

**Reply by dodalovicgran** [reply]
This seems to be a wezterm problem. [link](https://github.com/wez/wezterm/issues/3840#event-9528501151)
